tests/main/store-state/snap/meta/snap.yaml.in

```diff
@@ -1,6 +1,7 @@
 name: SNAPNAME
 summary: generic snap
 version: '1.0'
+base: core22
```

Comment:
Could we add this in the other pass later?

Reply:
@bboozzoo ?

Reply:
I'd prefer to avoid pulling in core because it masked bugs and wrong assumptions in the tests.